### PR TITLE
more consistent CentComm naming

### PR DESCRIPTION
**Pull request by strO0pwafel**
## About the PR
this pr changes a lot of the public-facing instances of "centcom" to"centcomm"

## Why / Balance
i wanted to start learning git and this seemed like an easy first pr. 

**Changelog**
:cl: 
- fix: Fixed inconsistent naming of CentComm.

### CODE REVIEW

**Comment by PJB3005** [comment]
CentCom is the correct spelling though.

**Comment by strO0pwafel** [comment]
https://github.com/space-wizards/space-station-14/pull/21108#issuecomment-1771962208 i got it from here

**Comment by 0x6273** [comment]
I think we should go with "CentComn" instead. An n is kinda like half of an m, so it's a good compromise that everyone will surely be happy with.

**Comment by PJB3005** [comment]
https://en.wikipedia.org/wiki/United_States_Central_Command

**Comment by HolySSSS** [comment]
We in command are saving up cents with 1 letter shorter name, think about the budget!

**Comment by ps3moira** [comment]
Why would CentCom have a second M?

**Comment by IProduceWidgets** [comment]
Everyone hush, I'm trying to talk on the centercom.

File: Resources/Locale/en-US/cargo/bounties.ftl [hunk 1]
```diff
@@ -95,18 +95,18 @@ bounty-description-mouse = Station 13 ran out of freeze-dried mice. Ship some fr
...
+bounty-description-spear = CentComm's security forces are going through budget cuts. You will be paid if you ship a set of spears.
 bounty-description-syringe = The NT anti-drug task force is in need of some syringes in order to distribute them to low-income communities. Help some keep some peoples' jobs secure.
 bounty-description-toolbox = There's an absence of robustness at Central Command. Hurry up and ship some toolboxes as a solution.
 bounty-description-tech-disk = The new research assistant on Station 13 spilled a soda on the RND server. Send them some technology disks so they can build up their recipes.
-bounty-description-trash = Recently a group of janitors have run out of trash to clean up, without any trash Centcom wants to fire them to cut costs. Send a shipment of trash to keep them employed, and they'll give you a small compensation.
+bounty-description-trash = Recently a group of janitors have run out of trash to clean up, without any trash Centcomm wants to fire them to cut costs. Send a shipment of trash to keep them employed, and they'll give you a small compensation.
```
Comment (Plykiya) [inline Resources/Locale/en-US/cargo/bounties.ftl:109]: CentComm

File: Resources/Prototypes/Entities/Clothing/Head/hoods.yml [hunk 2]
```diff
@@ -308,7 +308,7 @@
   parent: ClothingHeadHatHoodWinterBase
   id: ClothingHeadHatHoodWinterCentcom
   categories: [ HideSpawnMenu ]
-  name: Centcom winter coat hood
+  name: Centcomm winter coat hood
```
Comment (Plykiya) [inline Resources/Prototypes/Entities/Clothing/Head/hoods.yml:311]: CentComm

File: Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml [hunk 3]
```diff
@@ -580,7 +580,7 @@
   - type: Item
     heldPrefix: blue
   - type: IdCard
-    jobTitle: Centcom Agent
+    jobTitle: Centcomm Agent
```
Comment (Plykiya) [inline Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml:583]: CentComm

File: Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml [hunk 4]
```diff
@@ -785,7 +785,7 @@
   - type: Item
     heldPrefix: blue
   - type: IdCard
-    jobTitle: Centcom Quarantine Officer
+    jobTitle: Centcomm Quarantine Officer
```
Comment (Plykiya) [inline Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml:788]: CentComm

File: Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml [hunk 5]
```diff
@@ -1997,7 +1997,7 @@
   parent: VendingMachine
   id: VendingMachineCentDrobe
   name: CentDrobe
-  description: A one-of-a-kind vending machine for all your centcom aesthetic needs!
+  description: A one-of-a-kind vending machine for all your centcomm aesthetic needs!
```
Comment (Plykiya) [inline Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:2000]: CentComm

File: Resources/Prototypes/Objectives/traitor.yml [hunk 6]
```diff
@@ -38,7 +38,7 @@
 - type: entity
   parent: [BaseTraitorObjective, BaseLivingObjective]
   id: EscapeShuttleObjective
-  name: Escape to centcom alive and unrestrained.
+  name: Escape to centcomm alive and unrestrained.
```
Comment (Plykiya) [inline Resources/Prototypes/Objectives/traitor.yml:41]: CentComm

File: Resources/Prototypes/Objectives/thief.yml [hunk 7]
```diff
@@ -509,7 +509,7 @@
 - type: entity
   parent: [BaseThiefObjective, BaseLivingObjective]
   id: EscapeThiefShuttleObjective
-  name: Escape to centcom alive and unrestrained.
+  name: Escape to centcomm alive and unrestrained.
```
Comment (Plykiya) [inline Resources/Prototypes/Objectives/thief.yml:512]: CentComm

File: Resources/Prototypes/Objectives/traitor.yml [hunk 8]
```diff
@@ -84,7 +84,7 @@
 - type: entity
   parent: [BaseTraitorObjective, BaseKillObjective]
   id: KillRandomPersonObjective
-  description: Do it however you like, just make sure they don't make it to centcom.
+  description: Do it however you like, just make sure they don't make it to centcomm.
```
Comment (Plykiya) [inline Resources/Prototypes/Objectives/traitor.yml:87]: CentComm....

**Review by Plykiya — CHANGES_REQUESTED** [review]
at might as well as make the capitalization consistent as well

File: Resources/Prototypes/Entities/Clothing/Head/hoods.yml (hunk 2 above)
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Head/hoods.yml:311]:
```suggestion
  name: CentComm winter coat hood
```

**Comment by Plykiya** [comment]
wtf actually merged, that's crazy

**Pull request merged**